src/sql/workbench/browser/designer/designer.ts

```diff
@@ -836,6 +868,46 @@ export class Designer extends Disposable implements IThemable {
 					headerRowHeight: TableHeaderRowHeight,
 					editorLock: new Slick.EditorLock()
 				});
+				let buttonColumn: Slick.Column<Slick.SlickData>[] = [];
```

Comment:
remove this variable and move the canMoveRows handling block after the declaration of `columns` variable and add the move row column definition to it instead of introducing another collection. you can add it to the head of the columns array.

Reply:
I'm not sure what the point of it is. The move row plugin is supposed to be the first one, so what's wrong in setting the columns in order? I can instead instantiate the columns array with the moveRowPlugin and add the subsequent ones, if that's what you're asking for

Reply:
that works too.